Src/Newtonsoft.Json/Bson/BsonReader.cs

```diff
@@ -175,7 +175,7 @@ private string ReadElement()
         }
 
         /// <summary>
-        /// Reads the next JSON token from the stream.
+        /// Reads the next JSON token from the underlying <see cref="BinaryReader"/> or <see cref="Stream"/>.
```

Comment:
A BinaryReader can only wrap a stream so I don't think both need to be mentioned, just stream. Repeat in other comments that mention both.